com.cohesionforce.avro.gen.feature/feature.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature
+      id="com.cohesionforce.avro.gen.feature"
+      label="Feature"
+      version="1.0.0.qualifier"
+      provider-name="COHESIONFORCE">
+
+   <description url="http://www.example.com/description">
+      [Enter Feature Description here.]
+   </description>
+

```

Comment:
Need to add a description of the feature and add copyright and license information.